frontend/pages/schedule/ManageSchedulePage/components/ScheduleListWrapper/ScheduleListWrapper.tsx

```diff
@@ -77,7 +79,22 @@ const ScheduleListWrapper = (props: IScheduleListWrapperProps): JSX.Element => {
     );
   };
 
-  const tableHeaders = generateTableHeaders();
+  const onActionSelection = (
+    action: string,
+    global_scheduled_query: IGlobalScheduledQuery
+  ): void => {
+    switch (action) {
+      case "edit":
+        onEditScheduledQueryClick(global_scheduled_query);
+        break;
+      case "remove":
+        onRemoveScheduledQueryClick([global_scheduled_query.id]);
+        break;
+      default:
+    }
```

Comment:
Best practice is to fill the default with one of the cases.

Reply:
Good to know!